consensus/src/dag/order_rule.rs

```diff
@@ -135,13 +135,14 @@ impl OrderRule {
         self.lowest_unordered_anchor_round = anchor.round() + 1;
 
         let mut dag_writer = self.dag.write();
-        let ordered_nodes: Vec<_> = dag_writer
+        let mut ordered_nodes: Vec<_> = dag_writer
             .reachable_mut(&anchor, None)
             .map(|node_status| {
                 node_status.mark_as_ordered();
                 node_status.as_node().clone()
             })
             .collect();
+        ordered_nodes.reverse();
```

Comment:
why not use rev() on the iter?

Reply:
this is not a double iterator so it doesn't have rev()